students/17th/hkk/hyeonsookang/user/views.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+from django.http  import JsonResponse
+
+from django.views import View
+
+from user.models  import Id, Password
```

Comment:
```suggestion
import json

from django.http  import JsonResponse
from django.views import View

from user.models  import Id, Password
```
줄바꿈이 과합니다. 
임포트의 순서는 다음과 같습니다.
1. python  라이브러리
2. django모듈
3. 사용자가 만든 서브클래스 혹은 메소드 들 
위 순서로 정의 하시되 그룹지어 줄바꿈 해주세요.